common/lib/xmodule/xmodule/studio_editable.py

```diff
@@ -46,6 +45,16 @@ def get_preview_view_name(block):
         return AUTHOR_VIEW if hasattr(block, AUTHOR_VIEW) else STUDENT_VIEW
 
 
+class StudioEditableModule(StudioEditableBlock):
+    """
+    Helper methods for supporting Studio editing of xmodules.
+
+    This class is only intended to be used with an XModule, as it assumes the existence of
+    self.descriptor and self.system.
```

Comment:
Is this comment still true? StudioEditableBlock doesn't use self.descriptor.  It uses self.system, does that mean StudioEditableBlock should only be used with an XModule? I'm confused.

Also, you might want to explain why you have an empty class.  Why wouldn't `StudioEditableModule = StudioEditableBlock` work? Or why not just change all uses of StudioEditableModule to use StudioEditableBlock?